realm/realm-library/src/androidTestObjectServer/kotlin/io/realm/UserTests.kt

```diff
@@ -161,19 +160,16 @@ class UserTests {
 
     @Test
     fun logOutAsync_throwsOnNonLooperThread() {
-        val user: User = app.login(Credentials.anonymous())
         try {
-            user.logOutAsync { fail() }
+            anonUser.logOutAsync { fail() }
             fail()
         } catch (ignore: IllegalStateException) {
         }
     }
 
-    @Ignore("FIXME: Wait for linkUser support in ObjectStore")
     @Test
     fun linkUser() {
```

Comment:
Add more tests that link other credentials types and put them in different methods, e.g. `fun linkUser_userApiKey()`.